vbench/src/tests/timer/timer_test.cpp

```diff
@@ -1,17 +1,17 @@
 // Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
-#include <vespa/vespalib/testkit/test_kit.h>
+#include <vespa/vespalib/gtest/gtest.h>
 #include <vbench/test/all.h>
 #include <vespa/vespalib/util/time.h>
 
 using namespace vbench;
 
-IGNORE_TEST("timer") {
+TEST(TimerTest, ignored_timer_test) {
```

Comment:
Consider `DISABLED_` prefix instead of `ignored_` prefix if test should be disabled.

Reply:
I do not want to disable it